AirCasting/Graph/Graph.swift

```diff
@@ -79,22 +79,25 @@ struct Graph: UIViewRepresentable {
     }
     
     func getMidnightsPoints(startingDate: Date) -> [Double] {
-        let now = Date()
+        let day = Date()
```

Comment:
Should this always be `Date()`? When we are looking at the session which finished eg. in 2019 we are calculating all of the midnights since the first measurements, and that can be a lot. Do you think it would be possible to pass the lastMeasurementDate to this function?

Reply:
Cool, haven't thought about it but yeees. I will try if it is gonna work - if will I change a way of making it